### PR TITLE
Create `FundingService` script includes

### PR DESCRIPTION
**Pull request by juliusfitzhugh-ccpo**
Create script includes with `getFollowOnCLIN` method.

Ticket: AT-7849

### CODE REVIEW

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_3a155edd97ed11102b1ef0b0f053aff0.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+				// exclude current clin
+				return clinNumber !== clinRec.clin_number.toString(); } )
+            .filter(function(clinNumber) { 
+				// get clins that are in future periods
+                // assumes the clin first char identifies the period
+                var currentPeriod = clinRec.clin_number.toString()[0]++;
+                var selectedClinPeriod = clinNumber[0]++; 
+
+               // period ceiling is 9 currently
+               return selectedClinPeriod > currentPeriod && selectedClinPeriod <= 9;  })
```
Comment (juliusfitzhugh-ccpo) [inline f600233d1b154d507b782f84604bcb12/update/sys_script_include_3a155edd97ed11102b1ef0b0f053aff0.xml:50]: @jasonburkert-ccpo @jeffsegal-ccpo what are we considering a follow-on CLIN? 
- Is it any CLIN that is in a future period? (e.g.,`1XXX` -> `2XXX`) 
OR
- A CLIN that is in a future period and matches? (e.g., `1001` -> `2001`)

Comment (jeffsegal-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_include_3a155edd97ed11102b1ef0b0f053aff0.xml:50]: The latter. That is, 2020 is not a follow-on to 1017.

Comment (juliusfitzhugh-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_include_3a155edd97ed11102b1ef0b0f053aff0.xml:50]: That's what I was thinking, but wanted to make sure. I'll make that change. I'll also rebase and update the `Evaluate Time Remaining for CLIN` to include the script include.

Comment (juliusfitzhugh-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_include_3a155edd97ed11102b1ef0b0f053aff0.xml]: Updated `getFollowOnCLIN` to have matching CLINS as mentioned above. Changes in https://github.com/dod-ccpo/atat-snow/pull/127/commits/1784740f87deefe62ba3f32fcdae744bad31d436.

**Comment by juliusfitzhugh-ccpo** [comment]
Update `Get Effective Follow-on CLIN` action to include `getFollowOnCLIN` method

```javascript
 outputs.follow_on_clin = new FundingService().getFollowOnCLIN(inputs.clinRecord);
```

**Review by jeffsegal-ccpo — APPROVED** [review]
Looks good

**Pull request merged**